newsfragments/1226.misc.rst

```diff
@@ -0,0 +1,2 @@
+Change events on the event_bus from string to python enum
```

Comment:
No need to talk about this change in the release changelog given it doesn't affect the user (so an empty `1226.empty.rst` would be enough)

Reply:
fixed